bin/copy_to_distributed_table

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# default values for certain options
+format='text'
+schema='public'
+
+# we'll append to this string to build options list
+options='OIDS false, FREEZE false'
+
+# outputs usage message on specified device before exiting with provided status
+usage() {
+	cat << 'E_O_USAGE' >&"$2"
+usage: copy_to_distributed_table [-BCTHh] [-c encoding] [-d delimiter]
+	[-e escape] [-n null] [-q quote] [-s schema] filename tablename
+
+  B : use binary format for input
+  C : use CSV format for input
+  T : use text format for input
+  H : specifies file contains header line to be ignored
+  h : print this help message
+
+  c : specifies file is encoded using `encoding`
+      Default: the current client encoding
+  d : specifies the character used to separate columns
+      Default: a tab character in text format, a comma in CSV format
+  e : specifies the character used to escape quotes
+      Default: the same as the `quote` value (quotes within data are doubled)
+  n : specifies the string that represents a null value
+      Default: \N in text format, an unquoted empty string in CSV format
+  q : specifies the quoting character to be used when a data value is quoted
+      Default: double-quote
+  s : specifies the schema in which the target table resides
+      Default: 'public'
+E_O_USAGE
+
+	exit $1;
```

Comment:
I initially just had one argument (status) but realized that commands I consider "well-behaved" output to stdout (and exit 0) when you _ask_ them for help (i.e. pass `-h`) but output to stderr (with non-zero status) when you make some usage mistake. So that necessitated adding a second argument.

I agree that reversing the order makes more sense.